modules/API/external/external_api.js

```diff
@@ -81,6 +81,26 @@ function changeParticipantNumber(APIInstance, number) {
 }
 
 /**
+ * Generates array with URL params based on the passed config object that will
+ * be used for the Jitsi Meet URL generation.
+ *
+ * @param config {object} the config object.
+ * @returns {Array<string>} the array with URL param strings.
+ */
+function configToURLParamsArray(config) {
+    const params = [];
+    Object.keys(config).forEach( key => {
+        try {
+            params.push(key + '='
+                + encodeURIComponent(JSON.stringify(config[key])));
```

Comment:
Is the JSON.stringify intentional? Looks like that could add additional quotation marks around string parameters.